preview/Win7Msix/Tests/test.ps1

```diff
@@ -91,7 +91,7 @@ else
 
 ShowTestHeader("FindAllPackages")
 $output = & $executable  -FindAllPackages 
-if ($output -contains "notepadplus_0.0.0.1_x64__8wekyb3d8bbwe" -and -not ($output -contains "fakedoesnotexist_1.0.0.1_x64__8wekyb3d8bbwe"))
+if ($output -contains "notepadplus_0.0.0.0_x64__8wekyb3d8bbwe" -and -not ($output -contains "fakedoesnotexist_1.0.0.1_x64__8wekyb3d8bbwe"))

```

Comment:
>notepadplus_0 [](start = 23, length = 13)

did you also update the package in the folder then? doesn't show binary was changed.

Reply:
Yes updated the packages with MinVersion="6.1.7601.0", MaxVersionTested="10.0.10240.0"